packages/terra-framework-docs/src/terra-dev-site/test/menu/menu/DisabledMenuChildren.test.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Menu from 'terra-menu';
+import Button from 'terra-button';
+import IconCaretDown from 'terra-icon/lib/icon/IconCaretDown';
+
+class DisabledMenuChildren extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.handleRequestClose = this.handleRequestClose.bind(this);
+    this.setButtonNode = this.setButtonNode.bind(this);
+    this.getButtonNode = this.getButtonNode.bind(this);
+    this.state = { open: false };
+  }
+
+  componentDidMount() {
+    this.forceUpdate();
+  }
+
+  handleButtonClick() {
+    this.setState({ open: true });
+  }
+
+  handleRequestClose() {
+    this.setState({ open: false });
+  }
+
+  setButtonNode(node) {
+    this.buttonNode = node;
+  }
+
+  getButtonNode() {
+    return this.buttonNode;
+  }
+
+  render() {
+    return (
+      <div>
+        <Menu
+          isOpen={this.state.open}
+          targetRef={this.getButtonNode}
+          onRequestClose={this.handleRequestClose}
+        >
+          <Menu.Item text="Default Item 1" id="TestContent1" />
+          <Menu.Item text="Default Item 2" id="TestContent2" isDisabled />
```

Comment:
```suggestion
          <Menu.Item text="Default Item 2" id="TestContent2" isDisabled />
```
can you add onclick for disabled item to display alert . so that we can ensure that handlers for disabled items are not triggered on click or on keydown

Reply:
added in https://github.com/cerner/terra-framework/pull/1655/commits/9f203a7f6965b7a3e0cd0266d3fd9171e32694c2